### PR TITLE
Migrated TC test_ec_eager_lock_functional_validation.py

### PR DESCRIPTION
**Pull request by Sheetalpamecha**
[Migrated TC test_ec_eager_lock_functional_validation.py](https://github.com/gluster/glusto-tests/blob/master/tests/functional/disperse/test_ec_eager_lock_functional_validation.py)

Updates: #292
Signed-off-by: Sheetal Pamecha <spamecha@redhat.com>

### CODE REVIEW

**Comment by Sheetalpamecha** [comment]
```
Results for Special
Test nature : nonDisruptive
TestName : vol_create_test
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     disp    |     PASS    |        00:00:27       |      NA     |
|  dist-disp  |     PASS    |        00:00:29       |      NA     |
+-------------+-------------+-----------------------+-------------+
TestName : vol_destroy_test
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     disp    |     PASS    |        00:00:48       |      NA     |
|  dist-disp  |     PASS    |        00:00:57       |      NA     |
+-------------+-------------+-----------------------+-------------+
Results for disperse
Test nature : nonDisruptive
TestName : test_ec_eager_lock_functional_validation
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     disp    |     PASS    |        00:02:24       |      NA     |
|  dist-disp  |     PASS    |        00:02:45       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   0   |
|         Non Disruptive Tests         |   3   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   6   |
|            Tests Skipped             |   0   |
|             Total Tests              |   3   |
|   Disruptive Tests Pass Percentage   |   0   |
| Non Disruptive Tests Pass Percentage | 100.0 |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 06 minutes 14 seconds

```

File: tests/functional/disperse/test_ec_eager_lock_functional_validation.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        """
+
+        # Enable EagerLock
+        redant.set_volume_options(self.vol_name,
+                                  {'disperse.eager-lock': 'on',
+                                   'disperse.eager-lock-timeout': '30'},
+                                  self.server_list[0], True)
+
+        # Start profile on volume.
+        redant.profile_start(self.vol_name, self.server_list[0])
```
Comment (nik-redhat) [inline tests/functional/disperse/test_ec_eager_lock_functional_validation.py:92]: Considering we are starting profiling in this TC, better to make it as disruptive as profiling is itself a heavy operation.

**Review by nik-redhat — CHANGES_REQUESTED** [review]

File: tests/functional/disperse/test_ec_eager_lock_functional_validation.py (hunk 1 above)
Comment (Sheetalpamecha) [reply tests/functional/disperse/test_ec_eager_lock_functional_validation.py:92]: Done

**Comment by Sheetalpamecha** [comment]
```
Results for disperse
Test nature : disruptive
TestName : test_ec_eager_lock_functional_validation
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     disp    |     PASS    |        00:02:19       |      NA     |
|  dist-disp  |     PASS    |        00:02:51       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   2   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 07 minutes 23 seconds
```

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**